cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -551,20 +612,31 @@ def act_sigmoid_scaled(x):
         checkpoint_callback=ckpt_callback,
     )
 
+    # %%
+    # the Estimator hides the following details:
+    # 1. binding Spark DataFrames to a deep learning training script
+    # 2. reading data into a format that can be interpreted by the training framework
+    # 3. distributed training using Horovod
```

Comment:
We have to capitalize the first letter here as well. Sorry if this is confusing, but it just doesn't seem right to use the lowercase.